app/controllers/stats_controller.rb

```diff
@@ -163,10 +163,17 @@ def create_spreadsheet(report)
     axlsx
   end
 
+  def is_general_close_report_present?
+    close_report_scope = Pundit.policy_scope(current_user, ReportType.closed_cases_report)
+    close_report_scope.count > 0
```

Comment:
I think this could just be
```suggestion
    Pundit.policy_scope(current_user, ReportType.closed_cases_report).present?
```

Reply:
Also a tiny thing, I'm a little confused by the name of the method here. Would it make sense to call the method `closed_cases_report_present?` or are we distinguishing it from another closed cases report?

Reply:
@Polsonby , yes there is another version of closed report for offender-sar specifically which is R401. 
R007 is the one I call it as "General" which is used by other case types like FOI SAR,  I am not sure how to name it